### PR TITLE
Update content type

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
## Description

Update content type:

- [x] remove 'note that'
- [x] write shorter sentences

## Type of suggestion

* [x] Styleguide

## Product & version (if relevant)
Umbraco CMS v10, 12 & 13

### CODE REVIEW

File: 10/umbraco-cms/reference/management/models/contenttype.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ return contentType.Icon;
 
 ### .Id
 
-Gets the unique `ContentType` Id as a `Int`, this ID is based on a Database identity field, and is therefore not safe to reference in code which are moved between different instances, use Key instead.
+Gets the unique `ContentType` Id as an `Int`, this ID is based on a Database identity field. It's not safe for code referencing across different instances, use Key instead.
```
Comment (alina-tincas) [inline 10/umbraco-cms/reference/management/models/contenttype.md:165]: To keep the same wording as per other docs: https://github.com/umbraco/UmbracoDocs/pull/5674/files (same goes for v12 and v13 as well)
```suggestion
Gets the unique `ContentType` Id as an `Int`. The ID, derived from a database identity field, isn't safe for code references as they are moved across instances. Therefore it's recommended to use `Key` instead.
```

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Hi @erikjanwestendorp thank you for the PR! 

Hope you had nice holidays 😊 ¨

I have noticed for the ID section there was a similar PR where we went with different wording: https://github.com/umbraco/UmbracoDocs/pull/5674/files would you be up for using the same to keep text in the docs consistent? And if you agree, can you please make the change to all versions as suggested? 😊

File: 10/umbraco-cms/reference/management/models/contenttype.md (hunk 1 above)
Comment (erikjanwestendorp) [reply 10/umbraco-cms/reference/management/models/contenttype.md:165]: @alina-tincas Good point! 😄

File: 13/umbraco-cms/reference/management/models/contenttype.md [hunk 2]
```diff
@@ -176,7 +176,7 @@ return contentType.Key;
 
 ### .Level
 
-Gets or Sets the given `ContentType` level in the site hierarchy as an `Int`. ContentTypes placed at the root of the tree, will return 1, content right underneath will return 2, and so on.
+Gets the unique `ContentType` Id as an `Int`. The ID, derived from a database identity field, isn't safe for code references as they are moved across instances. Therefore it's recommended to use `Key` instead.
```
Comment (alina-tincas) [inline 13/umbraco-cms/reference/management/models/contenttype.md:179]: seems this change is not really relevant?

File: 13/umbraco-cms/reference/management/models/contenttype.md [hunk 3]
```diff
@@ -162,7 +162,7 @@ return contentType.Icon;
 
 ### .Id
 
-Gets the unique `ContentType` Id as a `Int`, this ID is based on a Database identity field, and is therefore not safe to reference in code which are moved between different instances, use Key instead.
+Gets the unique `ContentType` Id as an `Int`, this ID is based on a Database identity field. It's not safe for code referencing across different instances, use Key instead.
```
Comment (alina-tincas) [inline 13/umbraco-cms/reference/management/models/contenttype.md:165]:
```suggestion
Gets the unique `ContentType` Id as an `Int`. The ID, derived from a database identity field, isn't safe for code references as they are moved across instances. Therefore it's recommended to use `Key` instead.
```

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Nice! I also had a nice holiday however they passed by too fast 😆 

I have noticed you have made on v13 docs 1 change that is it not that relevant 🙈

**Comment by erikjanwestendorp** [comment]
@alina-tincas Holidays always go by too quickly 🙈. Oops my bad... 😅, just updated the PR.

**Review by alina-tincas — APPROVED** [review]

**Comment by alina-tincas** [comment]
Indeed, and no worries 😊

**Pull request merged**